### DISCUSSION TITLE
How to support Multisig Signing

### DISCUSSION THREAD

**Discussion by ManuelFerreras** [WalletConnect/walletconnect-monorepo#3790] [Developer Support]
Hey guys! We are building a Crypto Wallet and came into an issue. We've just migrated to Account Abstraction and we are not being able to make an off-chain signing method. For ex. we are not being able to connect and sign the initial OpenSea message correctly. Is there a workaround for this?

**Comment by glitch-txs** [comment]
Hi can you try here?
https://se-sdk-dapp.vercel.app/

**Reply by ManuelFerreras** [reply]
@glitch-txs unfortunately the only method that works for us is sending transactions. In all other methods we get JSON-RPC Request Rejected

**Reply by glitch-txs** [reply]
this should work with AA, make sure you set the methods as supported in the session proposal

**Reply by ManuelFerreras** [reply]
@glitch-txs we support all methods, but I suppose that this is related to the return signature being invalid. do you know which would be the best approach for AA signing? off-chain or on-chain?
